src/main/scala/wdltool/GraphPrint.scala

```diff
@@ -1,71 +1,101 @@
 package wdltool
 
 import java.nio.file.{Files, Paths}
+import java.util.concurrent.atomic.AtomicInteger
+
+import wdl4s.wdl.{CallOutput, Declaration, If, Scatter, WdlCall, _}
+import wdl4s.wdl.WdlGraphNode
 
-import wdl4s.{CallOutput, Declaration, If, Scatter, _}
 import scala.collection.JavaConverters._
 
+import cats._
+import cats.implicits._
+
 object GraphPrint {
 
-  case class WorkflowDigraph(workflowName: String, digraph: Set[String])
+  case class WorkflowDigraph(workflowName: String, digraph: NodesAndLinks)
+  case class NodesAndLinks(nodes: Set[String], links: Set[String])
+  implicit val scopesAndLinksMonoid = new Monoid[NodesAndLinks] {
```

Comment:
I submitted a PR to derive the monoid: https://github.com/broadinstitute/wdltool/pull/34